doc/api/assert.md

```diff
@@ -243,6 +263,7 @@ assert.fail(1, 2, 'whoops', '>');
 <!-- YAML
 added: v0.1.97
 -->
+* `value` {Number|String|Error}
```

Comment:
Any type would work with this function. The examples below are just that -- examples.